### PR TITLE
Specify package manager version for yarn (and bump patch)

### PR DESCRIPTION
**Pull request by timja**
Should fix https://github.com/jenkinsci/jenkins/pull/5824#issuecomment-1191120951

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/6895"></a>

This PR should also fix it but it's slightly bigger: https://github.com/jenkinsci/jenkins/pull/6894

### CODE REVIEW

**Review by NotMyFault — APPROVED** [review]

**Review by daniel-beck — APPROVED** [review]
Let's give it a try 🤷

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

----

(I plan to merge this once there's a green build)

**Comment by timja** [comment]
Didn't work but closer: https://github.com/jenkinsci/jenkins/pull/6892#issuecomment-1191536043

**Pull request merged**
